playbook/app/pb_kits/playbook/pb_tooltip/_tooltip.tsx

```diff
@@ -43,31 +45,34 @@ const Tooltip = (props: TooltipProps): React.ReactElement => {
     icon = null,
     interaction = false,
     placement: preferredPlacement = "top",
+    position,
```

Comment:
Hey @augustomallmann, can we change the name of this prop that you're creating so it's not 'position'? We're currently working on creating a global 'position' prop and once that it released this will conflict with that global prop. We're already looking into changing other kits that have a 'position' prop so let's make this different on the first iteration so we don't have to come back and change it!